packages/dashboards/src/lib/components/timeseries-widget/types.ts

```diff
@@ -23,7 +23,7 @@ export interface ITimeseriesWidgetSeries {
 
 export interface ITimeseriesWidgetData<T = ITimeseriesWidgetSeriesData> {
     id: string;
-    name: string;
+    name?: string; // unused
```

Comment:
@lukasholcik, I set this property to optional because it doesn't appear to be used anywhere. If it's optional, consumers can use the interface without having to set the name unnecessarily.